app/views/layouts/application.html.erb

```diff
@@ -72,6 +72,7 @@
               <div class="header__popup__nav-links">
                 <%= link_to t('.header.dashboard'), dashboard_path, class: "header__nav-link" %>
                 <%= link_to t('.header.edit_profile'), edit_profile_path, class: "header__nav-link" %>
+                <%= link_to t('.header.edit_settings'), edit_settings_path, class: "header__nav-link" %>
```

Comment:
What would you think of naming this "Edit security settings"? It might be more distinctive, as you could argue that your Twitter account is also a "setting".

Reply:
That will work, but maybe it's a bit too long and might look kind of awkward in the drop down menu. However, "Security settings" without the "Edit" could be an option.
Anyway, I guess the exact wording will be decided by the maintainers.